### PR TITLE
Add KSP2 Inflator

### PR DESCRIPTION
**Pull request by techman83**
This adds a secondary inflator and queue for ksp2 inflations, along with adding the relevant GAME env var to them.

- No changes to existing inflator infra, beyond exposing the GAME environment variable
- Secondary inflator will sit dormant until the rest of the pieces are added

This should allow us to get KSP-CKAN/CKAN#3797 merged

### CODE REVIEW

File: prod-stack.py [hunk 1]
```diff
@@ -754,6 +760,27 @@
...
+            ('GAME', 'KSP')
+        ],
+        'volumes': [
+            ('ckan_cache', '/home/netkan/ckan_cache')
+        ]
+    },
+    {
+        'name': 'InflatorKsp2',
+        'image': 'kspckan/inflator',
+        'memory': '256',
```
Comment (HebaruSan) [inline prod-stack.py:772]: Looks like the original Inflator had `156` before #224, we could probably start at least that low here to save memory given that both repos are currently <200KiB and the downloads are at least that tiny with no part or planet packs yet:

```
$ du -sh KSP2*
153K    KSP2-CKAN-meta
130K    KSP2-NetKAN
```

**Review by HebaruSan — APPROVED** [review]
Pending merge of KSP-CKAN/CKAN#3797 and consideration of comment about RAM, this is what we need as far as I understand it.

File: prod-stack.py (hunk 1 above)
Comment (techman83) [reply prod-stack.py:772]: True, though it's the max memory they're allowed to use rather then allocated. There is just shy of 900MB headroom for it, so we should be ok.

Comment (HebaruSan) [reply prod-stack.py:772]: Works for me!

**Pull request merged**